apps/frontend/src/discover/__tests__/Discover.test.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import nock from 'nock';
+import { render, waitFor } from '@testing-library/react';
+import { authTestUtils } from '@asap-hub/react-components';
+import { MemoryRouter, Route } from 'react-router-dom';
+import Discover from '../Discover';
+
+import { API_BASE_URL } from '../../config';
+
+const renderHome = async () => {
+  const result = render(
+    <authTestUtils.Auth0Provider>
+      <authTestUtils.WhenReady>
+        <authTestUtils.LoggedIn user={{}}>
+          <MemoryRouter initialEntries={['/']}>
+            <Route exact path="/" component={Discover} />
+          </MemoryRouter>
+        </authTestUtils.LoggedIn>
+      </authTestUtils.WhenReady>
+    </authTestUtils.Auth0Provider>,
+  );
+  await waitFor(() =>
+    expect(result.queryByText(/auth0/i)).not.toBeInTheDocument(),
+  );
+  await waitFor(() =>
+    expect(result.queryByText(/loading/i)).not.toBeInTheDocument(),
+  );
+  return result;
+};
+
+test('renders discover header', async () => {
+  nock(API_BASE_URL, {
+    reqheaders: { authorization: 'Bearer token' },
+  })
+    .get('/discover')
+    .once()
+    .reply(200, {
+      members: [],
+      pages: [],
+      aboutUs: '',
+    });
+
+  const { getByText } = await renderHome();
+  expect(getByText(/discover/i, { selector: 'h1' })).toBeVisible();
+});
+
+test('renders dicover with guidance, about and members', async () => {
+  nock(API_BASE_URL, {
+    reqheaders: { authorization: 'Bearer token' },
+  })
+    .get('/discover')
+    .once()
+    .reply(200, {
+      aboutUs: '<h1>About us</h1>',
```

Comment:
Now that Tom has started with `@asap-hub/fixtures`, would it be possible to source this example data there (while of course object spreading and inserting the data points relevant to the test in here)?

Reply:
They would also add typing (whether in fixtures or not), which is helpful for future schema changes

Reply:
We already have a lot of data spread over many tests and stories that will take a long time to centralize, so would appreciate not adding more data types with example data all over the place, that's why I'm asking to do this straight away from now on :sweat_smile:

Reply:
Looks like the Discover story has much more example data for this data type, we should probably use that for the fixture :)

Reply:
I'm changing the page card in a follow up PR. I'll do this work on that PR :+1: ?

Reply:
Okay, yeah. It might actually reduce the size of this PR by avoiding duplication, but not sure. Make sure to catch all places where you've added this type of data though :sweat_smile: 
And can you add the Response type annotation here (`{...} as BlaResponse`) so it's safe at least